README.md

```diff
@@ -30,7 +30,7 @@ const limiter = new RedisRateLimiter({
   client: redisClient, // client instance from `redis` or `ioredis`
   namespace: 'rate-limiter', // prefix for redis keys
   interval: 60000, // milliseconds
-  maxInInterval: 10,
+  maxInInterval: 5,
```

Comment:
Can you explain why this is a typo?

Reply:
Hi, apologise for the lack of detail here. 

Later on, in **Configuration Options**, when referring to the code example it references 5 requests per minute but I believe the example provided would allow for 10.

```
For example, if you want a user to be able to perform 5 actions per minute, this should be 60000.
```

```
maxInInterval: number - The number of actions allowed in each interval. For example, in the scenario above, this would be 5
```

Reply:
Ah, good catch! Thanks.